contrib/camel/config/400-camelsource.yaml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    eventing.knative.dev/source: "true"
+  name: camelsources.sources.eventing.knative.dev
```

Comment:
I am wondering if the kind should just be `camel.sources.eventing.knative`. What do you think? (Removing the `sources.sources`)

Reply:
Well, thinking long-term I'd keep `CamelSource` here, because I expect we're going to implement using Camel other pieces in the future.

E.g. outside knative we already implement other kind of resources, like EIP or "outbound connectors", but Knative defines only the concept of source currently (i.e. "inbound connector"). I expect those objects to be different and so each one having a different CRD.

I think that field affects the way you interact with the CLI: `k get camelsource` is more meaningful than `k get camel`...